src/ussd_registration.js

```diff
@@ -224,6 +224,10 @@ go.app = function() {
                 question: $(questions[name]),
                 check: function(content) {
                     if (go.utils.is_valid_msisdn(content)) {
+                        if (self.im.user.answers.state_msisdn_father == self.im.user.answers.state_msisdn_father) {
+                            // submit 'father_only' as msg receiver
+                            self.im.user.set_answer('state_msg_receiver', 'father_only'); //?
+                        }
```

Comment:
Rather remove this, will sort this out with the actual submission of the registration.